packages/extension-app/package.json

```diff
@@ -13,16 +13,13 @@
   },
   "devDependencies": {
     "@open-wc/webpack-import-meta-loader": "^0.4.1",
-    "@types/chrome": "^0.0.91",
-    "@types/firefox-webext-browser": "^70.0.1",
     "copy-webpack-plugin": "^5.1.1",
     "webpack": "^4.41.5",
     "webpack-cli": "^3.3.10",
     "webpack-extension-manifest-plugin": "^0.5.0"
   },
   "dependencies": {
     "extensionizer": "^1.0.1",
-    "web-ext-types": "^3.2.1",
-    "webextension-polyfill": "^0.6.0"
```

Comment:
why remove this? did you just replace it with `extensionizer.d.ts` instead?

Reply:
There are 2 ways to doing things:

1.
```ts
 import 'webextension-polyfill';

browser.runtime.connect(); // Firefox/WebExt syntax but it works on chrome too, because of the polyfill
```

2. 
```ts
import extensionizer from 'extensionizer';

extensionizer.runtime.connect(); // Works on chrome & firefox
```

(related to https://github.com/paritytech/substrate-light-ui/pull/714#discussion_r367548359). Initially i added 1 because it's what most people use I guess. But in this project we use 2, so i'm deleting 1.

Reply:
ahh that makes sense, thanks for the clarification!